.github/workflows/build-cli.yml

```diff
@@ -88,21 +88,21 @@ jobs:
       - name: Download Artifact
         uses: actions/download-artifact@v3
         with:
-          name: Linux-GNU-eludris
+          name: Linux-GNU-derailed
 
       - name: Create Package Files
         run: |
           mkdir -p .debpkg/usr/bin
           mkdir -p .rpmpkg/usr/bin
-          chmod +x eludris
-          cp -p eludris .debpkg/usr/bin/
-          cp -p eludris .rpmpkg/usr/bin/
+          chmod +x derailed
+          cp -p derailed .debpkg/usr/bin/
+          cp -p derailed .rpmpkg/usr/bin/
       - name: Build DEB
         uses: jiro4989/build-deb-action@v2
         with:
           package: ${{ env.NAME }}
           package_root: .debpkg
-          maintainer: Oliver Wilkes <oliver@eludris.gay>
+          maintainer: Oliver Wilkes <oliver@derailed.gay>
```

Comment:
So who's gonna buy this domain <img src="https://cdn.discordapp.com/emojis/980418824975319040.webp?size=160&quality=lossless" width=21>

Reply:
skl